### DISCUSSION TITLE
Retrieved scheduled search results (csv/json)

### DISCUSSION THREAD

**Discussion by waschoo** [CrowdStrike/falconpy#1042] [Q&A]
Hi,
https://www.falconpy.io/Service-Collections/Scheduled-Reports.html
-> Possible to also retrieved result from "scheduled search" with this ?  (crowdstrike.com/scheduled-search)
I try to find a way to export data from FQL query:
https://www.reddit.com/r/crowdstrike/comments/ssk5w1/retrieve_scheduled_search_results_csv_or_json_via/

Another question, there is something in the roadmap to retrieved data directly from Logscale API ?

Thanks :)

**Comment by jshcodes** [comment]
Hi @waschoo -

Correct! You are able to retrieve scheduled search results using the [Schedule Reports](https://falconpy.io/Service-Collections/Scheduled-Reports.html) service collection. 

LogScale support is currently being investigated.  Stay tuned... 😃

**Comment by waschoo** [comment]
HI Josh,
Thanks :)
Good news for Logscale !
About "schedule reports", yes it works well.  (Dashboards and reports -> Scheduled reporting) 
But I'd like to do the same for  "Investigate scheduled search" (Investigate -> Scheduled search"), no idea if it's possible (?)
